<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCCurve</class>
 <widget class="QWidget" name="DCCurve">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="lista_partidas">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>256</width>
     <height>261</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="isWrapping" stdset="0">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="titulo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="text">
    <string> Partidas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_seleccionar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Seleccionar Partida</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_nueva_partida">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>120</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nueva Partida</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_nombre">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>190</y>
     <width>231</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="text_nombre">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>180</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_crear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Crear</string>
   </property>
  </widget>
  <widget class="QLabel" name="text_bienvenida">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>20</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="espacio_nombre">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="espacio_titulo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="espacio_bienvenida">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="button_actualizar">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <zorder>espacio_nombre</zorder>
  <zorder>espacio_bienvenida</zorder>
  <zorder>espacio_titulo</zorder>
  <zorder>lista_partidas</zorder>
  <zorder>titulo</zorder>
  <zorder>button_seleccionar</zorder>
  <zorder>button_nueva_partida</zorder>
  <zorder>edit_nombre</zorder>
  <zorder>text_nombre</zorder>
  <zorder>button_crear</zorder>
  <zorder>text_bienvenida</zorder>
  <zorder>button_actualizar</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
